### ISSUE TITLE
[a11y]: MultiSelect - The descendent <div> element with "option" role has no focusable child element

### ISSUE THREAD

**Issue by ghost** [carbon-design-system/carbon#11341]
### Package

carbon-components-react

### Browser

Chrome, Firefox

### Operating System

MacOS

### Package version

v7.52.x (~7.52.0)

### React version

v17.x.x (^17.0.2)

### Automated testing tool and ruleset

IBM Equal Access Accessibility Checker - Latest Deployment

### Description

1. Go to the [MultiSelect default storybook](https://react.carbondesignsystem.com/?path=/story/components-multiselect--default)
2. Open the MultiSelect (use inspect element on one of the entry to open DevTools without closing the menu)
3. Scan with IBM Equal Access Accessibility Checker (while the MultiSelect is opened)
4. You get the following accessibility violation:
   - `2.1.1 Keyboard: The descendent <div> element with "option" role has no focusable child element`

The error comes from the fact that the div that encompasses the different values has `tabindex="-1"` and the children elements don't define `tabindex` so they are considered not focusable by the accessibility checker.

Note: I'm still able to navigate through the elements using keyboard.

![Screenshot 2022-05-04 at 10 58 51](https://user-images.githubusercontent.com/51360096/166651405-62e8892b-3418-44f5-a6e9-4aab41ba5ed6.png)

### WCAG 2.1 Violation

Criterion 2.1.1: Keyboard

### CodeSandbox example

https://react.carbondesignsystem.com/?path=/story/components-multiselect--default

### Steps to reproduce

I'm not able to reproduce from the CodeSandbox, I don't understand why... but the storybook does have the same error we have.

1. Go to the [MultiSelect default storybook](https://react.carbondesignsystem.com/?path=/story/components-multiselect--default)
2. Open the MultiSelect (use inspect element on one of the entry to open DevTools without closing the menu)
3. Scan with IBM Equal Access Accessibility Checker (while the MultiSelect is opened)
4. You get the following accessibility violation:
   - `2.1.1 Keyboard: The descendent <div> element with "option" role has no focusable child element`

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/f555616971a03fd454c0f4daea184adf41fff05b/.github/CODE_OF_CONDUCT.md)
- [X] I checked the [current issues](https://github.com/carbon-design-system/carbon/issues) for duplicate problems

**Comment by tay1orjones** [comment]
@tombrunet We are seeing the error on this, but is this a bug with the checker? The items are still navigable, though it is of note the activedescendant attribute isn't placed on the children until they're hovered/focused. Maybe this is why the checker is failing this state?

**Comment by Laureen-Ginier** [comment]
Hello @tay1orjones and @tombrunet, have you any idea whether this issue is a false positive?
We lowered the severity of the bug on our side as the items are still navigable using keyboard. But it is monitored and cannot be closed as is.
Thanks

**Comment by tombrunet** [comment]
I think the issue is that the listbox is in the accessibility tree with no content when the dropdown is closed. When closed, that listbox element should be visibility:hidden.

**Comment by dakahn** [comment]
@tombrunet even with the listbox in the accessibility tree--if its not accessible via keyboard or screen reader isn't it as good as hidden?
